hibike/flash.sh

```diff
@@ -1,10 +1,57 @@
-#/bin/bash
+#!/bin/bash
 
+NAME="$0"
+SENSOR_TYPES="$(ls devices/)"
```

Comment:
Thanks for the feedback. Apparently it's bad form to rely on the output of `ls` anyways, so I will change this script to use your script, and your script to use globs.